webui/react/src/pages/F_ExpList/ComparisonView.tsx

```diff
@@ -101,7 +101,7 @@ const ComparisonView: React.FC<Props> = ({
           message="No experiments selected."
         />
       ) : (
-        <Pivot items={tabs} />
+        <Pivot destroyInactiveTabPane items={tabs} />
```

Comment:
https://github.com/determined-ai/determined/blob/db01c03feecd73c8aed6aab873c5d8523921d3ef/webui/react/src/components/ParallelCoordinates.tsx#L24

`containerRef.current?.clientHeight` and `containerRef.current?.clientWidth` becomes 0 when moving to different tab from the hyperparameter tab. That caused the issue since width and height should be more than 0px.


<img width="1200" alt="image" src="https://github.com/determined-ai/determined/assets/109113616/22df6c4d-01e4-474b-ad5a-3524e1118043">

Reply:
Will this reintroduce the issue of the data reloading every time you switch tabs? Also please leave a comment explaining this in the code.

Reply:
This does not reintroduce the issue of the data reloading every time user switches tabs. 
Without `destroyInactiveTabPane`, parallel coordinates height and width become 0px when other tabs are selected. That violates the `hermes-parallel-coordinates` rule.

https://github.com/determined-ai/determined/assets/109113616/2fe31214-dec5-4a5e-97bd-ae8422c7a4a0

---

Or other solution is that changing `hermes-parallel-coordinates` to accept 0px width and height. 
As the author of library @hkang1, which solution would be better? 

https://github.com/hkang1/hermes/blob/c8fc72faa367a598e449f04012fd8c90ace8d788/src/index.ts#L51

Reply:
Throwing of an error when the container of the chart is 0x0 might be too harsh. I do avoid calculations and drawing if the size is 0x0 already so it might be enough to just warn the user in the console.

Reply:
Thanks. Let me know when the new version is published, then i'll revert the current change and update the lib version

Reply:
There is a new version published @ [0.16.7](https://www.npmjs.com/package/hermes-parallel-coordinates/v/0.6.17) now that avoids throwing an error when the chart container is of size 0px (width or height)

Reply:
Thank you for updating the lib quickly. Pushed the new commit. Could you review the PR?